boxoffice/messageclient.py

```diff
@@ -0,0 +1,18 @@
+import requests
+
+from baseframe import _
+
+from . import app, rq
+
+
+@rq.job('boxoffice')
+def send_telegram_message(buyer_fullname, line_item_title):
+    with app.test_request_context():
+        message_text = _(f'{buyer_fullname} purchased {line_item_title}')
+        send_text = f'https://api.telegram.org/bot{app.config["TELEGRAM_BOT_TOKEN"]}/sendMessage'
+        params = {
+            'chat_id': app.config['TELEGRAM_CHAT_ID'],
+            'message_thread_id': app.config['TELEGRAM_MESSAGE_THREAD_ID'],
```

Comment:
This is optional, so use `app.config.get('TELEGRAM_THREADID')`

Note: the telegram key names we're using in Funnel are `TELEGRAM_APIKEY`, `TELEGRAM_CHATID` and `TELEGRAM_THREADID`.